### ISSUE TITLE
The Schema should contain ApplyToExecutable for the parameter

### ISSUE THREAD

**Issue by theJasonHelmick** [PowerShell/Crescendo#162]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/Microsoft.PowerShell.Crescendo)
- [X] Search the existing issues.

### Steps to reproduce

see issue: https://github.com/PowerShell/Crescendo/issues/30

### Expected behavior

```console
See issue: https://github.com/PowerShell/Crescendo/issues/30
```

### Actual behavior

```console
See issue: https://github.com/PowerShell/Crescendo/issues/30
```

### Error details

```console
Object model currently supports ApplyToExecutable - missing from Schema. Tests are already included.
```

### Environment data

```PowerShell
See issue: https://github.com/PowerShell/Crescendo/issues/30
```

### Version

1.0

**Comment by JamesWTruher** [comment]
fixed in #161

**Comment by theJasonHelmick** [comment]
Thank you! Fixed/closed - in release: https://www.powershellgallery.com/packages/Microsoft.PowerShell.Crescendo/1.1.0-RC1
